src/www/ui_components/add-server-view.js

```diff
@@ -299,9 +306,11 @@ Polymer({
     input.toggleClass('input-invalid', input.invalid);
     if (input.invalid) {
       this.$.addServerFooter.hidden = true;
+      this.$.addServerFooterAlt.hidden = true;
       this.$.invalidAccessKeyFooter.hidden = false;
     } else {
-      this.$.addServerFooter.hidden = false;
+      this.$.addServerFooter.hidden = false || this.showAltAccessMessage;
```

Comment:
That aligns with my suggestion. A variable that indicates whether to show the invalid key error, or the server suggestion message. 👍🏼

Reply:
Done.

Added `invalidAccessKeyInput` property. I know negative booleans are bad, but the logical split here is (no access key input, valid access key input) vs (invalid access key input) so not using the negative made the logic more confusing.

Unfortunately no boolean logic except `!var` is allowed in html, so switching to dom-if required adding two new computed properties and two new functions to contain the boolean logic. I've switched the name of the passed property to `useAltAccessMessage` everywhere to avoid confusion with the `show...` properties that are being computed in this element.